lib/logstash/outputs/kusto.rb

```diff
@@ -106,6 +106,15 @@ class LogStash::Outputs::Kusto < LogStash::Outputs::Base
   # starts processing them in the main thread (not healthy)
   config :upload_queue_size, validate: :number, default: 30
 
+  # Host of the proxy , is an optional field. Can connect directly
+  config :proxy_host, validate: :string, required: false
+
+  # Port where the proxy runs , defaults to 80. Usually a value like 3128
+  config :proxy_port, validate: :number, required: false , default: 80
+
+  # TODO check Proxy URL can be over http or https. Dowe need it this way or ignore this & remove this
```

Comment:
It can be over http or https, no other options (that we support anyway), so you can remove the TODO